README.md

```diff
@@ -329,8 +329,7 @@ For more details, see [Adding Package Dependencies to Your App](https://develope
     <img src="/Docs/Images/BetterSafariViewDemo-watchOS.png" width="275">
 </p>
 
-You can see how it works on each platform and compare it with the other naive implementations in the demo project. Check out the demo app by opening BetterSafariView.xcworkspace.
-
+You can see how it works on each platform and compare it with the other naive implementations in the demo project. Check out the demo app by opening BetterSafariViewDemo.xcodeproj, adding the BetterSafariView package dependency in Xcode (`File` > `Add Packages` > `Add Local` > select the root folder of BetterSafariView), and finally run build.
```

Comment:
```suggestion
You can see how it works on each platform and compare it with the other naive implementations in the demo project. Check out the demo app by opening BetterSafariViewDemo.xcworkspace adding the BetterSafariView package dependency in Xcode (`File` > `Add Packages` > `Add Local` > select the root folder of BetterSafariView > Select Build Targets to add to), and finally run build.
```

This should remain .xcworkspace because that is the only file available. Devs will also need to select a build target  as shown here <img width="1411" alt="Screenshot 2024-01-01 at 13 38 22" src="https://github.com/stleamist/BetterSafariView/assets/384288/ef98d3fc-e287-4cb9-99f5-d155f1d0ed73">